netbox/dcim/choices.py

```diff
@@ -332,6 +340,12 @@ class PowerPortTypeChoices(ChoiceSet):
             (TYPE_NEMA_620P, 'NEMA 6-20P'),
             (TYPE_NEMA_630P, 'NEMA 6-30P'),
             (TYPE_NEMA_650P, 'NEMA 6-50P'),
+            (TYPE_NEMA_1030P, 'NEMA 10-30P'),
+            (TYPE_NEMA_1050P, 'NEMA 10-50P'),
+            (TYPE_NEMA_1420P, 'NEMA 14-20P'),
+            (TYPE_NEMA_1430P, 'NEMA 14-30P'),
+            (TYPE_NEMA_1450P, 'NEMA 14-50P'),
+            (TYPE_NEMA_1460P, 'NEMA 14-60P'),
         )),
         ('NEMA (Locking)', (
             (TYPE_NEMA_L515P, 'NEMA L5-15P'),
```

Comment:
We should add the locking variants

Reply:
I was using a chart I found online for my verification that was missing some L variants.  This has been resolved.  All but the 10-50 have an Locking variant and were added to the choices.